src/examples/javascript.stories.js

```diff
@@ -0,0 +1,69 @@
+import { storiesOf } from '@storybook/html';
+import copyCodeBlock from '../copyCodeBlock';
+import { customStyles } from './customHtml';
+import { usageExample, usageExampleJsHighlight } from '../helpers';
+import hljs from 'highlight.js/lib/highlight';
+
+// Register languages for hljs
+hljs.registerLanguage('css', require('highlight.js/lib/languages/css'));
+hljs.registerLanguage('javascript', require('highlight.js/lib/languages/javascript'));
+
+const a11yLightStyle = require('!url-loader!highlight.js/styles/a11y-light.css');
+const draculaStyle = require('!url-loader!highlight.js/styles/dracula.css');
+
+const jsExample = `
+    import { Pants } from 'fancy-things';
+
+    var cool = 'cool';
+    let stuff = 'stuff';
+    const things = 'things';
+
+    function coolStuffAndThings(a, b, c) {
+        return \`\$\{a\}\ \$\{b\}\ and \$\{c\}\`;
+    }
+
+    coolStuffAndThings(cool, stuff, things);
+    Pants();
+`;
+
+storiesOf('Javascript', module)
+    .add('Default', () => `
+        <link rel="stylesheet" href="${draculaStyle}">
```

Comment:
Is there a reason you're putting an `hljs` style in the stories that aren't running through `hljs`?

Reply:
Because I wanted to differentiate the JS code block from the example code